### PR TITLE
PR for #1474: Update high resoultion media query

### PR DESCRIPTION
**Pull request by drublic**
A PR for #1474.

Please keep the discussion within #1474.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+* Update high resoultion media query
```
Comment (arthurvr) [inline CHANGELOG.md]: resoultion ->resolution

Comment (drublic) [inline CHANGELOG.md]: Thanks. Updated.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+* Update high resolution media query
```
Comment (arthurvr) [inline CHANGELOG.md]: We will need the `### HEAD` thing [like we did before](https://github.com/h5bp/html5-boilerplate/commit/1c93f0a0569231c820013042c2c19102a32d1693#diff-4ac32a78649ca5bdd8e0ba38b7006a1eL1).

Comment (alrra) [inline CHANGELOG.md]: > We will need the ### HEAD thing like we did before.

`### HEAD ` was added by another commit, so we're good!

**Comment by alrra** [comment]
Merged with some modification in https://github.com/h5bp/html5-boilerplate/commit/0d9f00826ac4a0154b6f2ed7ad0c127a124e3c34. Thanks @drublic!

**Pull request closed without merging**